src/main/kotlin/no/nav/familie/ef/sak/brev/BrevmottakereService.kt

```diff
@@ -36,6 +42,59 @@ class BrevmottakereService(val brevmottakereRepository: BrevmottakereRepository)
         }
     }
 
+    fun hentBrevnottakereForFagsak(fagsakId: UUID): BrevmottakereDto? {
+        return frittståendeBrevmottakereRepository.findByFagsakIdAndSaksbehandlerIdent(
+            fagsakId,
+            SikkerhetContext.hentSaksbehandler(),
+        )?.let { BrevmottakereDto(personer = it.personer.personer, organisasjoner = it.organisasjoner.organisasjoner) }
+    }
+
+    fun lagreBrevmottakereForFagsak(fagsakId: UUID, brevmottakereDto: BrevmottakereDto): UUID {
+        validerAntallBrevmottakere(brevmottakereDto)
+        validerUnikeBrevmottakere(brevmottakereDto)
+
+        val eksistrendeBrevmottakere =
+            frittståendeBrevmottakereRepository.findByFagsakIdAndSaksbehandlerIdent(
+                fagsakId,
+                SikkerhetContext.hentSaksbehandler(),
+            )
+
+        return if ((eksistrendeBrevmottakere != null)) {
+            oppdaterBrevmottakere(eksistrendeBrevmottakere, fagsakId, brevmottakereDto)
+        } else {
+            opprettBrevmottakere(fagsakId, brevmottakereDto)
+        }.fagsakId
+    }
+
+    fun slettBrevmottakereForFagsakOgSaksbehandlerHvisFinnes(fagsakId: UUID, saksbehandlerIdent: String) =
+        frittståendeBrevmottakereRepository.findByFagsakIdAndSaksbehandlerIdent(fagsakId, saksbehandlerIdent)?.let {
+            frittståendeBrevmottakereRepository.deleteById(it.id)
+        }
+
+    private fun opprettBrevmottakere(
+        fagsakId: UUID,
+        brevmottakereDto: BrevmottakereDto,
+    ) = frittståendeBrevmottakereRepository.insert(
+        BrevmottakereFrittståendeBrev(
```

Comment:
Bare for lesbarhet - kunne tatt med saksbehandlerIdent? Tror jeg er enig med deg her - beholde som default, bare tenker litt høyt.

Reply:
Vi diskuterte denne muntlig og kom fram til å beholde den som den er.